html-css/week1/Vadim/OrderProject/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./files/css/styles.css">
+        <title>ORDER FORM</title>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+    </head>
+
+    <body>
+        <!-- Header -->
+        <header class="order">
+            <div class="order__summary">
+                <h1>Order Summary</h1>
+            </div>
+            <div class="order__total">
+                <div class="order__totalpermonth">
+                    <p>Total</p>
+                    <p>$333 / month</p>
+                </div>
+                <hr class="order__line">
+                <div class="order__totalnow">
+                    <p>Total</p>
+                    <p>$100</p>
+                </div>
+            </div>
+        </header>
+
+        <!-- Payment Method -->
+        <section class="payment">
+            <h2 class="payment__method">Payment Method</h2>
+            <div class="payment__option">
+                <input type="radio" name="select" id="option-1" class="payment__radio" checked />
+                <input type="radio" name="select" id="option-2" class="payment__radio" />
+                <label for="option-1" class="option option-1">
+                    <img class="payment__img1" src="./files/images/cards.webp" alt="Pay with Credit or Debit Cards" />
+                    <span>Credit / Debit</span>
+                </label>
+                <label for="option-2" class="option option-2">
+                    <img class="payment__img2" src="./files/images/paypal.webp" alt="Pay with Paypal" />
+                    <span>PayPal</span>
+                </label>
+            </div>
+        </section>
+
+        <!-- Card Data Input -->
+        <form class="card">
+            <h2 class="card__info">Card Info</h2>
+            <article class="card__input">
+                <div class="card__data">
+                    <label for="card_number" class="card__text">Card number</label>
+                    <input class="card__number" name="card_number" id="card__number" type="text" pattern="\s*(\S\s*){16,}" maxlength="19" placeholder="1234 1234 1234 1234" required>
```

Comment:
Clicking the label doesn't focus the input field as in the other input fields you have. This is because `card_number` is with a single underline for `for` and the `name` attribute but with 2 __ for the `id`.

```suggestion
                    <label for="card__number" class="card__number">Card number</label>
                    <input class="card__number" name="card__number" id="card__number" type="text" pattern="\s*(\S\s*){16,}" maxlength="19" placeholder="1234 1234 1234 1234" required>
```
Be lazy and never type variables you already have written - just copy paste (:

Reply:
:D That's on me, plus being dyslectic doesn't really help while typing!